pkg/volume/gce_pd/gce_util.go

```diff
@@ -149,11 +150,59 @@ func (mounter *gceSafeFormatAndMount) Mount(source string, target string, fstype
 	}
 	args = append(args, options...)
 	args = append(args, source, target)
-	glog.V(5).Infof("exec-ing: /usr/share/google/safe_format_and_mount %v", args)
-	cmd := mounter.runner.Command("/usr/share/google/safe_format_and_mount", args...)
+	safeFormatAndMountPath := "/usr/share/google/safe_format_and_mount"
+	glog.V(5).Infof("exec-ing: %s %v", safeFormatAndMountPath, args)
+	cmd := mounter.runner.Command(safeFormatAndMountPath, args...)
 	dataOut, err := cmd.CombinedOutput()
 	if err != nil {
-		glog.V(5).Infof("error running /usr/share/google/safe_format_and_mount\n%s", string(dataOut))
+		glog.V(5).Infof("error running %s\n%s", safeFormatAndMountPath, string(dataOut))
+		// if safe_format_and_mount is not available on this system use linux utilities directly.
+		if strings.Contains(err.Error(), safeFormatAndMountPath+": no such file or directory") {
+			return mounter.formatAndMount(source, target, fstype, options)
+		}
+	}
+	return err
+}
+
+// Uses unix utils to format and mount the given disk
+func (mounter *gceSafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
+
+	// Add the default options from safe_format_and_mount
+	options = append(options, "discard")
```

Comment:
Yeah I just copied the script here. I did a bit of research and I have not found great reviews of the "discard" option. I will remove it unless you say otherwise.